### DISCUSSION TITLE
Tile Card: Cover-position for Awning

### DISCUSSION THREAD

**Discussion by gcosta74** [home-assistant/frontend#21145] [Dashboard]
For awnings the cover-position needs to be inverted. Currently if you use the tile card for an awning the feature cover-position will result in a weird slider view.
While opening the awnings for example 20 % the cover-position  slider will show it as it would be 80% open.
For Awnings the slider needs to be inverted:

![cover_pos_inverted](https://github.com/home-assistant/frontend/assets/18614022/bb3d608a-23a1-4efb-bed8-03410feda415)

**Comment by Misiu** [comment]
Ref: https://github.com/home-assistant/frontend/pull/17687
I had a PR, but it was rejected.
Maybe with more user voices the feature can be reconsidered.
